taskRequests/details/style.css

```diff
@@ -182,7 +182,6 @@ body {
 .requestors {
   padding: 1rem;
   align-self: flex-start;
-  border-left: solid 1px rgba(0, 0, 0, 0.1);
```

Comment:
- Can you add a video with multiple requestor. 
- There's also no mention of why requestor css is changed in description.

Reply:
This border was removed when rendering github issue. As the issue card also have border. The border is applied when requettype is  ASSIGNMENT

other css were minor changes in mobile ui of requestors so will intoduced them in next PR to the linked issue